frontend/src/routes/+layout.svelte

```diff
@@ -1,13 +1,13 @@
 <script lang="ts">
-  import '../../app.css';
+  import '../app.css';
   import { staticSettings } from '@openvaa/app-shared';
   import { FeedbackModal } from '$lib/components/feedback/modal';
   import { Loading } from '$lib/components/loading';
   import { initLayoutContext } from '$lib/contexts/layout';
   import { t } from '$lib/i18n';
   import { openFeedbackModal } from '$lib/stores';
   import { MaintenancePage } from '$lib/templates/maintenance';
-  import type { LayoutData } from './$types';
+  import type { LayoutData } from './[[lang=locale]]/$types';
```

Comment:
If the `+layout.server.ts` is also moved to the same folder, shouldn't we import the types from './$types' as well?